app/models/patient.rb

```diff
@@ -35,12 +36,10 @@ class Patient < ApplicationRecord
                                                   'Other',
                                                   nil, ''] }
 
-  validates :monitoring_plan, inclusion: { in: ['None',
-                                                'Daily active monitoring',
-                                                'Self-monitoring with public health supervision',
-                                                'Self-monitoring with delegated supervision',
-                                                'Self-observation',
-                                                nil, ''] }
+  validates :monitoring_plan, inclusion: {
```

Comment:
I think we need more time to verify with app users before we add validation (which will be affected by this), so you can you add the `:api` context as @ngfreiter did in his PR that added validation to the Patient model?

Reply:
The validation on `monitoring_plan` already existed for the core app. This is just a slight refactor to how that's done so that it doesn't duplicate the already-defined enum (in `ValidationHelper`), and adds a better error message that the API will use.

Reply:
You're right - my mistake. Thanks!